Assets/UI/Panels/citypanel.xml

```diff
@@ -1,192 +1,192 @@
+                            <!-- <CheckBox ID="ProduceWithGoldCheck" Size="44,53" ButtonTexture="UnitPanel_ActionButton" ButtonSize="44,53" CheckTexture="CityPanel_ActionGoldProduction" CheckTextureOffset="0,32" CheckSize="32,32" CheckOffset="5,8" UnCheckTexture="CityPanel_ActionGoldProduction" UnCheckSize="32,32" UnCheckOffset="5,8" UseSelectedTextures="1" ToolTip="LOC_HUD_CITY_PURCHASE_WITH_GOLD" /> -->
+                            <!-- <CheckBox ID="ProduceWithFaithCheck" Size="44,53" ButtonTexture="UnitPanel_ActionButton" ButtonSize="44,53" CheckTexture="CityPanel_ActionFaithProduction" CheckTextureOffset="0,32" CheckSize="32,32" CheckOffset="5,8" UnCheckTexture="CityPanel_ActionFaithProduction" UnCheckSize="32,32" UnCheckOffset="5,8" UseSelectedTextures="1" ToolTip="LOC_HUD_CITY_PURCHASE_WITH_FAITH" /> -->
+                            <CheckBox ID="ChangeProductionCheck" Size="44,53" ButtonTexture="UnitPanel_ActionButton" ButtonSize="44,53" CheckTexture="CityPanel_ActionProduction" CheckTextureOffset="0,32" CheckSize="32,32" CheckOffset="5,8" UnCheckTexture="CityPanel_ActionProduction" UnCheckSize="32,32" UnCheckOffset="6,9" UseSelectedTextures="1" ToolTip="LOC_HUD_CITY_CHANGE_PRODUCTION">
+                                <Tutorial ID="TutOpenProduction" Style="TutorialContainer" Anchor="C,T" AnchorSide="O,O" Offset="0,50" TriggerBy="TutorialOpenProduction">
+                                    <Grid Style="TutorialCalloutGrid">
+                                        <Label Style="TutorialHeaderText" String="LOC_META_3_HEAD" />
+                                        <Label Style="TutorialBodyText" String="LOC_META_3_BODY" />
+                                    </Grid>
+                                    <SlideAnim Start="0,0" EndOffset="0,-20" Cycle="Bounce" Function="OutQuad" Anchor="C,B">
+                                        <Image Texture="Tutorial_ArrowV" Offset="-22,-20" Size="44,58" />
+                                    </SlideAnim>
+                                </Tutorial>
+                            </CheckBox>
+                        </Stack>
+                    </Grid>
+
+                </Stack>
+
+                <Button ID="CitizensGrowthButton" Offset="28,116" Texture="SelectionPanel_StatLarge" Size="41,42" Disabled="1">
+                    <Image Texture="CityPanel_CitizenIcon" Size="32,32" Anchor="C,C" Offset="0,-2" Color="255,255,255,150">
+                        <Label ID="PopulationNumber" Anchor="C,C" Style="CityPanelNumLarge" WrapWidth="70" String="999" Align="Center" KerningAdjustment="-1" Offset="-1,0" />
+                    </Image>
+                </Button>
+            </GridButton>
```

Comment:
This is an actual fix - the opening tag to this is at line 15 and this end tag here </GridButton> matches the basegame CityPanel.xml